### PR TITLE
update docs on inclusion of fallback in dispatch list summaries

### PR DESCRIPTION
**Pull request by naftali-g**
Naming convention:
 - PRs for features that are in design should have the "proposal" label
 - PRs for features that haven't landed yet should have the "future" label
 - PRs for upcoming releases should have the "release" label
 - PRs with new documentation for existing features should have the "existing feature" label

Before requesting review:
 - Make sure there is a ticket in the DOC board in Jira
 - Make sure CI is passing
   - Spell check failure may require adding backticks around code or updating `spelling_wordlist.txt`
   - See `README.md` for information about style and markdown syntax
   - If the CI Details link gives a 404, you need to log in to readthedocs.com
 - Add link to generated documentation here
   - you can find this by following the read the docs link from the CI check
 - Ask for help in #documentation

Jira ticket: TODO
Link to generated documentation: TODO

### CODE REVIEW

File: docs/cvl/methods.md [hunk 1]
```diff
@@ -203,58 +203,59 @@ methods {
...
    Example: `C.foo(uint)`
-2. Wildcard contract - a pattern specifying the function signature to match 
+2. Wildcard contract - a pattern specifying the function signature to match
    this signature on all available contracts (including the primary contract).
    Example: `_.bar(address)`
-3. Wildcard function - a pattern specifying a contract, and matches all 
-   external functions in specified contract.
+3. Wildcard function - a pattern specifying a contract, and matches all
+   external functions in specified contract (This pattern will also include the
+   contract's fallback if it's implemented).
```
Comment (johspaeth) [inline docs/cvl/methods.md:241]: This is the only change I could find. Next time please try to commit without the other line break changes.

Is it easy for you to revert? Then please only add what's required.

**Review by johspaeth — APPROVED** [review]
LGTM - as far I see it, you've only added 1 line which is sufficient for the change we implemented. Only request, please try to avoid committing the line breaks as it make it difficult to review.

File: docs/cvl/methods.md (hunk 1 above)
Comment (naftali-g) [reply docs/cvl/methods.md:241]: yes this is the only change.

Comment (johspaeth) [reply docs/cvl/methods.md:241]: I wasn't aware of this feature! Thanks for pointing it out to me.

**Pull request merged**